website/source/docs/internals/telemetry.html.md

```diff
@@ -50,742 +50,1468 @@ The following is an example telemetry dump snippet:
 [2017-12-19 20:38:00 +0000 UTC][S] 'vault.merkle.flushDirty': Count: 100 Min: 0.006 Mean: 0.024 Max: 0.098 Stddev: 0.019 Sum: 2.386 LastUpdated: 2017-12-19 20:38:09.848158309 +0000 UTC m=+10473.691527099
 ```
 
-You'll note that log entries are prefixed with the metric type as follows:
-
-- **[C]** is a counter
-- **[G]** is a gauge
-- **[S]** is a summary
-
-
 The following sections describe available Vault metrics. The metrics interval can be assumed to be 10 seconds when manually triggering metrics output using the above described signals.
 
 ## Internal Metrics
 
 These metrics represent operational aspects of the running Vault instance.
 
-### vault.audit.log_request
-
-**[S]** Summary (Milliseconds): Duration of time taken by all audit log requests across all audit log devices
-
-### vault.audit.log_response
-
-**[S]** Summary (Milliseconds): Duration of time taken by audit log responses across all audit log devices
+<table class="table table-bordered table-striped">
```

Comment:
Isn't there markdown syntax for tables? Or is it inadequate?

Reply:
There is a table syntax yes, but it is inadequate (or more precisely, it was judged inadequate by whoever created the Consul telemetry docs, and I am trying to reformat the vault docs so that they are consistent with Consul.)

Reply:
Ok, another thing to consult with Yoko on I guess.

Reply:
I switched us over to regular table syntax